torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -256,10 +256,32 @@ NvrtcFunction nvrtcCompile(
   // Major and minor is determined by device properties and
   // possibly "downcompiled" to a lower (compatible) compute architecture
   // based on the NVRTC version
-  const int major = prop->major;
-  const int minor = prop->minor;
+  int major = prop->major;
+  int minor = prop->minor;
   nvrtcProgram program;
 
+  if (nvrtc_major < 10 ||
+      (nvrtc_major == 10 &&
+       nvrtc_minor == 0)) { // clamp at sm_72 for CUDA < 10.1
+    if (major > 7 || (major == 7 && minor > 2)) {
+      major = 7;
+      minor = 2;
+    }
+  } else if (
+      nvrtc_major < 11 ||
+      (nvrtc_major == 11 &&
```

Comment:
If this was true, this would mean that you guys never run tests on cuda 11 + A100. Is that true?

Reply:
IIRC, we don't run CI on 11.0.
This is at least true per nvrtc doc, as well as the issue reported in #48200

Reply:
Granted that the bug reported is prior to the fix... So maybe it does fix the issue.
But I'm still quite worried that we go against the documentation.

Reply:
Let me reach out to nvrtc folks...
I don't have a card on sm_80 to test it, but from the message prompt by 3090, seems like it's working and this is a documentation issue....

Reply:
Shouldn't the nightly binaries with `cudatoolkit=11.0` also fail for the fuser test in this case on a A100 (which works)?

Reply:
Yes they should fail. #48200 does complain about invalid arch, but that's because arch is 86 which is legitimately not supported by 11.0.

Reply:
OK, it's documentation issue. :sigh
11.0 should support sm_80. same for 10.0 which support sm_75 as well. I'll fix this.